javascript.md

```diff
@@ -55,7 +60,11 @@ let bar = thing; // throws, undefined
 Rust:
 
 ```rust
-// TODO
+if true {
```

Comment:
Not including () is considered idiomatic. The compiler even warns you if you do include them.